data/json/furniture.json

```diff
@@ -1733,6 +1733,7 @@
     "type": "furniture",
     "id": "f_canvas_wall",
     "name": "canvas wall",
+    "description": "The canvas mesh enclosing a tent.",
```

Comment:
I don't think mesh is a great term, it makes one think of a porous cloth. I'd suggest something simpler here, like:
`A wall made of stretched, waterproof cloth`